release-content/release-notes/meshlet-bvh-culling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Virtual Geometry BVH culling
+authors: ["@SparkyPotato", "@atlv24"]
+pull_requests: [19318]
+---
+
+(TODO: Embed example screenshot here)
+
+Bevy's virtual geometry has been greatly optimized with BVH-based culling, leading to almost true scene-complexity invariance on the GPU.
+
+This gets rid of the previous cluster limit that limited the world to 2^24 clusters (about 4 billion triangles).
+There are now *no* hardcoded limits to scene size, only unique instance limits due to VRAM usage (since streaming is not yet implemented),
+and total instance limits due the current architecture requiring all instances to be uploaded to the GPU every frame.
```

Comment:
```suggestion
There are now *no* hardcoded limits to scene size. In practice you will only be limited by asset VRAM usage (since streaming is not yet implemented),
and total instance count due the current code requiring all instances to be re-uploaded to the GPU every frame.
```